fpdf/line_break.py

```diff
@@ -449,7 +465,7 @@ def add_character(
             self.fragments.append(Fragment("", graphics_state, k, url))
         active_fragment = self.fragments[-1]
 
-        if character == SPACE:
+        if character in [SPACE] + OTHER_SPACE_SYMBOLS:
```

Comment:
You probably want to turn this into a global list, eg. `BREAKING_SPACE_SYMBOLS` at the top, instead of concenating the same two lists twice in every iteration of the loop.